### PR TITLE
Update UG introduction, about, navigation and glossary sections

### PR DESCRIPTION
**Pull request by yl-ang**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -3,28 +3,50 @@ layout: page
...
-LibTask is a **desktop application for librarians to manage book loans and requests by patrons, optimized for use via a Command Line Interface (CLI)** while still having the benefits of a Graphical User Interface (GUI). If you can type fast, LibTask can get your book tracking tasks done faster than traditional GUI apps.
+LibTask is a **desktop application for librarians to manage book loans and requests by patrons, optimized for use via a [Command Line Interface (CLI)](#6-glossary)** while still having the benefits of a Graphical User Interface (GUI). If you can type fast, LibTask can get your book tracking tasks done faster than traditional GUI apps.
 
 * Table of Contents
 {:toc}
 
 --------------------------------------------------------------------------------------------------------------------
 
-## Introduction
+# **1. Introduction**
```
Comment (Halpfrog) [inline docs/UserGuide.md:13]: Good job with ensuring the numbers are correct

File: docs/UserGuide.md [hunk 2]
```diff
@@ -3,28 +3,50 @@ layout: page
...
+
+## 1.2. About the User Guide
+The purpose of this guide is to explain to you how to use **LibTask** and to allow you to have an enjoyable experience. It explains the features and commands compatible with LibTask.  It is also structured in a way that lets you easily navigate to relevant sections.
+
+Take note of following symbols that are used in this document:
+
+* :information_source:: This symbol is used to highlight extra knowledge.
+
+* :exclamation:: This symbol is used to warn users of potential actions that might corrupt the LibTask's data file.
```
Comment (Halpfrog) [inline docs/UserGuide.md:25]: Nice way to show the reader the purpose of the UG

File: docs/UserGuide.md [hunk 3]
```diff
@@ -3,28 +3,50 @@ layout: page
...
+
+For the list of definitions of vocabulary words used in this document can be found in our [Glossary](#6-glossary) section.
 
 --------------------------------------------------------------------------------------------------------------------
 
-## Quick start
+# **2. Quick start**
 
-1. Ensure you have Java `11` or above installed in your Computer.
+1. Ensure you have Java `11` or above installed in your Computer. You can follow this [guide](https://docs.oracle.com/en/java/javase/11/install) to download and install java.
```
Comment (Halpfrog) [inline docs/UserGuide.md:43]: Nice way to give the reader easy access to the required java version

**Review by aditi2313 — APPROVED** [review]
Looks good! Nicely done 👍

**Pull request merged**